### PR TITLE
Removing FIPS

### LINKED ISSUE #1772: FIPS enabled images break azure-file storage class
With the recent updates to the vm image and enabling FIPS mode in the kernel the CIFS mounts have stopped working.

After searching through logs and manual attempts to perform the mount I discovered this in the logs:
```
Jul 02 19:09:23 compute-1562081346-000000 kernel: CIFS VFS: could not allocate crypto hmacmd5
Jul 02 19:09:23 compute-1562081346-000000 kernel: CIFS VFS: could not crypto alloc hmacmd5 rc -2
Jul 02 19:09:23 compute-1562081346-000000 kernel: CIFS VFS: Error -2 during NTLMSSP authentication
Jul 02 19:09:23 compute-1562081346-000000 kernel: CIFS VFS: Send error in SessSetup = -2
Jul 02 19:09:23 compute-1562081346-000000 kernel: CIFS VFS: cifs_mount failed w/return code = -2
```

A quick internet search brought back multiple articles responding to the inquiry:
https://access.redhat.com/solutions/256053

It appears that ntlm uses md5 hashes which are not supported under FIPS mode.  The only supported mode is `sec=krb5`.  I'm not sure enabling krb5 is possible when using the storage class.

Doc: https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-file

This includes mount options but I'm not sure how we could hook up kerberos for the storage class:
https://github.com/feiskyer/kubernetes-handbook/blob/master/en/troubleshooting/azurefile.md

### PR DESCRIPTION
**Pull request by kwoodson**
Removing FIPS mode to fix CIFS mounts.

Fixes #1772 

```release-note
None
```

### CODE REVIEW

**Comment by kwoodson** [comment]
@mjudeikis if this works I went ahead and removed the `exit` bug from the script.sh.  Feel free to close #

**Comment by kwoodson** [comment]
Poor CI: `fatal: unable to access 'https://github.com/openshift/openshift-azure.git/': Could not resolve host: github.com`

File: pkg/vmimage/data/script.sh [hunk 1]
```diff
@@ -210,10 +209,10 @@ rpm -q atomic-openshift-node --qf '%{VERSION}-%{RELEASE}.%{ARCH}' >/var/tmp/open
 
 # Image hardening
 # Setup fips inside of grub
-dracut -f
-sed -i -re 's/GRUB_CMDLINE_LINUX="(.*)"$/GRUB_CMDLINE_LINUX="\1 fips=1"/' /etc/default/grub
-grub2-mkconfig -o /boot/grub2/grub.cfg
-grub2-mkconfig -o /boot/efi/EFI/redhat/grub.cfg
+# dracut -f
```
Comment (mjudeikis) [inline pkg/vmimage/data/script.sh]: A comment would be good or to remove them at all

**Comment by mjudeikis** [comment]
/lgtm 
lets fix this as we now stuck for 2 w ;/

**Comment by kwoodson** [comment]
Please @openshift-ci-robot, merge this quick before something else fails.

**Comment by charlesakalugwu** [comment]
> Please @openshift-ci-robot, merge this quick before something else fails.

haha that was awesome! :dancing_men:

**Pull request merged**